Lib/webbrowser.py

```diff
@@ -15,14 +15,15 @@ class Error(Exception):
 
 _browsers = {}          # Dictionary of available browser controllers
 _tryorder = []          # Preference order of available browsers
+_preferred_browser = None # The preferred browser for the current environment
```

Comment:
My original implementation did this, but I didn't find the enhancement useful. There weren't any ripples.

That said, I'll make the change if this is considered a blocker.